source/_integrations/homeworks.markdown

```diff
@@ -37,4 +37,18 @@ Send a custom command to the Lutron Homeworks controller.
 | Service data attribute | Optional | Example                 | Description                                         |
 | ---------------------- | -------- | ----------------------- | --------------------------------------------------- |
 | `controller_id`        | No       | `homeworks`             | The controller to which the command should be sent to. |
-| `command`              | No       | `KBP, [02:08:02:01], 1` | The command you want to send. This can either be a single command or a list of commands. |
+| `command`              | No       | `KBP, [02:08:02:01], 1` | The command you want to send. This can either be a single command or a list of commands. In addition to the commands supported by the controller, the special command `DELAY <ms>` is supported, where ms is the number of milliseconds to sleep. |
+
+#### Sending a list of commands 
+
+The example shows how to send `KBP`, wait 0.5 seconds, then send `KBR` to simulate a keypad button keypress with a duration of a half second.
+
+```yaml
+service: homeworks.send_command
+data:
+  controller_id: "homeworks"
+  command:
+    - "KBP, [02:08:02:01], 1"
```

Comment:
What does this mean? What's the command syntax? Where can the user find what commands are available?

Reply:
The command set is documented here: https://assets.lutron.com/a/documents/hwi%20rs232%20protocol.pdf, should we link to it?